docs/development/development_details.rst

```diff
@@ -7,6 +7,17 @@ Development Details
 This document contains somewhat-independent sections that provide details on
 topics covered in the :ref:`contributing_quickstart` guide.
 
+.. _dev-dependencies:
+
+A quick way to set up a complete development environment is to install astropy asdf-astropy
```

Comment:
The asdf-astropy looks out of place. Is it intentional?
```suggestion
A quick way to set up a complete development environment is to install astropy
```

Reply:
oh my, auto-correct has gone completely wrong here, I meant to type "as follow" 😅